interfaces/builtin/network_manager.go

```diff
@@ -67,6 +67,29 @@ network packet,
 
 /dev/rfkill rw,
 
+# Needed for modem connections using PPP
+/usr/sbin/pppd ix,
+/etc/ppp/** rwix,
+/dev/ppp rw,
+/dev/tty[^0-9]* rw,
+/run/udev/data/* r,
+/proc/sys/kernel/modprobe r,
+/bin/kmod ix,
+capability sys_module,
+/run/lock/*tty[^0-9]* rw,
+/run/ppp* rw,
```

Comment:
It is part of the core snap and interacts quite heavily with the kernel, so not sure if we should do that